commit 2b2cc6543c44beeb8331dc8e1a7eff2e071a8cc8
Author: Antony Polukhin <antoshkka@gmail.com>
Date:   Sun May 10 21:53:27 2015 +0300

    Fix error with missing include, that was found by Maarten de Vries

diff --git a/include/boost/variant/detail/element_index.hpp b/include/boost/variant/detail/element_index.hpp
index bd80bdd..842b018 100644
--- a/include/boost/variant/detail/element_index.hpp
+++ b/include/boost/variant/detail/element_index.hpp
@@ -16,6 +16,7 @@
 #include "boost/variant/recursive_wrapper_fwd.hpp"
 #include "boost/variant/variant_fwd.hpp"
 
+#include "boost/type_traits/remove_cv.hpp"
 #include "boost/mpl/find_if.hpp"
 
 namespace boost { namespace detail { namespace variant {
